.circleci/select_tests.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -o errexit
```

Comment:
maybe add `set -o pipefail`

Reply:
I don't think it can work. The point of this is that pytest may fail, and when it fails the output should be printed to stdout. With `pipefail` the script will exit as soon as pytest fails, no?

Reply:
Hm [according to the docs](https://www.gnu.org/software/bash/manual/html_node/Pipelines.html) (last paragraph) it only makes sure there are no swallowed non-zero exit codes anywhere within the pipe, which isn't the default.

Reply:
done